mugoh/qstioner-v2
# #163448826 Update POST Data to have Defaults for non-Required Values

#### What does this PR do?
- Create defaults for post request data
#### Description of Task to be completed?
- A user doesn't have to specify an `isadmin` value
- RSVP responses are posted in the JSON body
- JSON responses are in lowercase

#### How should this be manually tested?

1. Clone this repo into your machine
``` shell
$ git clone https://github.com/hogum/qstioner-v2.git
```
2. Navigate to the repo directory
``` shell
$ cd qstioner-v2
```
3. Setup a virtual environment and install the project requirements
``` shell
$ pip3 install virtualenvwrapper
```
 ``` shell
$ source virtualenvwrapper.sh
```
``` shell
$ mkvirtuanenv pyenv && pip install -r requirements.txt 
```
4. Checkout this Chore branch:
``` shell
$ git checkout ch-default-user-163448826
```
5. Test the endpoints:
 a)  `api/v1/auth/register/<int:question_id>`
- Registration should be possible without having to specify an `isadmin` value
b) `api/v1/meetups/meetup_id/response`
- Sending an RSVP should be possible by a POST request in the JSON body

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[163448826](https://www.pivotaltracker.com/story/show/163448826)
